dev-libs/libratbag/libratbag-0.13.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit meson python-single-r1 udev
+
+DESCRIPTION="Library to configure gaming mice"
+HOMEPAGE="https://github.com/libratbag/libratbag"
+SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc elogind systemd test"
+REQUIRED_USE="
+	${PYTHON_REQUIRED_USE}
+	^^ ( systemd elogind )
+"
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	virtual/pkgconfig
+	doc? ( app-doc/doxygen )
+	test? ( dev-util/valgrind )
+"
+RDEPEND="
+	${PYTHON_DEPS}
+	dev-libs/libevdev
+	virtual/libudev
+"
+DEPEND="
+	${RDEPEND}
+	dev-lang/swig
```

Comment:
dev-lang/swig certainly should be in `BDEPEND`.

Reply:
Fixed